README.md

```diff
@@ -1,10 +1,10 @@
 QupZilla Web Browser
 ----------------------------------------------------------------------------------------
 
-[![Build Status](https://travis-ci.org/QupZilla/qupzilla.svg?branch=master)](https://travis-ci.org/QupZilla/qupzilla)  
-Homepage: [http://www.qupzilla.com](http://www.qupzilla.com)  
-Blog: [http://blog.qupzilla.com](http://blog.qupzilla.com)  
-IRC: `#qupzilla` at `irc.freenode.net`  
+[![Build Status](https://travis-ci.org/QupZilla/qupzilla.svg?branch=master)](https://travis-ci.org/QupZilla/qupzilla)
+Homepage: [http://www.qupzilla.com](http://www.qupzilla.com)
+Blog: [http://blog.qupzilla.com](http://blog.qupzilla.com)
+IRC: `#qupzilla` at `irc.freenode.net`

```

Comment:
Btw who's spec is that? I've never seen that in CommonMark nor Daring Fireballs... but since this is a new discovery I may have missed it. Is it GH specific?